test/java/seedu/addressbook/data/AddressBookTest.java

```diff
@@ -162,6 +162,19 @@ public void clear() throws Exception {
         assertTrue(isEmpty(defaultAddressBook.getAllTags()));
     }
 
+    @Test
+    public void sort() throws Exception {
+        // Setup address book
+        emptyAddressBook.addPerson(bobChaplin);
```

Comment:
Going further you can consider moving all the steps necessary for the test to run (or preconditions) to a setup method before beginning the tests; correspondingly can put all the tear down in a method after the tests.